docs/philosophy/outcomes/platform_adoption.md

```diff
@@ -48,50 +60,188 @@ Measures a team’s ability to onboard into a new platform, tool or pattern and
 
     Expressed as a _timestamp_
 
-`Adoption end event(aE)`
+_Adoption end events ( $a_{E}$ )_
 
-:   Developer/application/team does something meaningful using the tool
+:   Developer/application/team does something valuable using the tool
 
     For example:
 
-    - Application deployed in X environment
-    - Code committed to X branch
-    - Workspace spun up
+    - Deploys an application to a specific environment
+    - Completes a story
+    - Commits some code to a specific branch
+    - Logs a certain number of hours of active use
 
     Expressed as a _timestamp_
 
 #### Formulas
 
-`aLT = aE - aS`
+_Adoption Lead Time ( $LT$ )_
+
+:   For any individual adoption event, the adoption lead time $a_{LT}$ can be calculated as follows:
+
+    $$
+    LT = a_{E} - a_{S}
+    $$
 
-### :fontawesome-solid-arrow-trend-up: __Adoption rate__
+**_Average Adoption Lead Time ( $\overline{x}LT$ )_**
 
-Measures the ability of the platform or tool to scale to support multiple teams and products
+:   Given a collection of $N$ individual adoption lead times ( $LT_{1}..LT_{N}$ ), the _average adoption lead time_ $\overline{x}LT$ can be calculated as follows:
 
-#### Variables
+    $$
+    \overline{x}LT = \frac{sum(LT_{1}..LT_{N})}{N}
+    $$
```

Comment:
It would be better to use the sum symbol? `\[\overline{x}LT = \frac{\sum_{1}^{N}(LT_{i})}{N}\]`